docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,681 @@
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: backend-ns
+      default: $conf1 # 'conf1' is going to be applied to all consumers of the backend service
+---
+# Frontend Service Owner creates a consumer policy 
+metadata:
+  namespace: frontend-ns
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      kuma.io/service: frontend_frontend-ns_svc_8080
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: backend-ns
+      default: $conf2 # 'conf2' refines 'conf1' that was already provided by the producer
+```
+
+We can formally define what's producer and consumer Kuma policies:
+
+* if for each `idx` the value of `spec.to[idx].targetRef.namespace` is equal to `metadata.namespace` then it's a producer policy
+* for each `idx` the value of `spec.to[idx].targetRef.namespace` is **not** equal to `metadata.namespace` then it's a consumer policy
+
+The concept of producer/consumer policies is applied only to policies with `to` array.
+Other policies, i.e. with `from` or without `from/to` are neither producer nor consumer policies.
+For convenience when referring to such type of policies we're going to call them **workload-owner policies**.
+The examples of workload-owner policies are MeshTrace, MeshMetric, MeshProxyPatch, MeshTrafficPermission, etc. 
+
+Top-level targetRef in Kuma policies always references a group of sidecars that are subjected to the change.
+New namespace-scoped policies are no exception, but we're going to implicitly add `k8s.kuma.io/namespace` and `kuma.io/zone` for **consumer** and **workload-owner policies**.
+
+```yaml
+# a valid consumer policy
+metadata:
+  namespace: frontend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  targetRef:
+    kind: Mesh # despite it targets 'Mesh' the policy will be applied to pods in the 'frontend-ns' of 'zone-1'
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: backend-ns
+---
+# workload-owner policy with 'from'
+metadata:
+  namespace: frontend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  targetRef:
+    kind: Mesh # despite it targets 'Mesh' the policy will be applied to pods in the 'frontend-ns' of 'zone-1'
+  from:
+    - targetRef:
+        kind: Mesh
+```
+
+### Validation
+
+Policies with `to` array that have items for local and non-local namespaces shouldn't be allowed:
+
+```yaml
+# invalid policy example
+metadata:
+  namespace: finance-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: finance-ns # this indicates the policy is a producer policy
+    - targetRef:
+        kind: MeshService
+        name: redis
+        namespace: redis-ns # this indicates the policy is a consumer policy
+```
+
+Policies that support both `from` and `to` and specify them at the same time shouldn't be allowed as well.
```

Comment:
This is a breaking change no? Is it namespaced policies that use from and to or policies using namespaced entities (routes, services, meshexternalServices)?